core/src/main/java/com/linecorp/armeria/common/stream/CancellableStreamMessage.java

```diff
@@ -195,6 +195,10 @@ boolean withPooledObjects() {
             return withPooledObjects;
         }
 
+        boolean notifyCancellation() {
+            return notifyCancellation;
+        }
```

Comment:
Could rename to `shouldNotifyCancellation` to avoid confusion - `notifyCancellation()` sounds like it will notify something.

Reply:
Thanks, updated. 😉